reya-rust-sdk/src/data_types.rs

```diff
@@ -1,9 +1,66 @@
 use alloy::primitives::Address;
 use alloy::primitives::I256;
 use alloy::primitives::U256;
+use alloy::sol;
+use dotenv::dotenv;
+use std::env;
+use url::Url;
+
+///
+/// configuration struct for the sdk
+///
+///
+#[derive(PartialEq, Eq, Clone, Debug)]
+pub struct SdkConfig {
+    pub order_gateway_contract_address: String,
+    pub passiv_perp_contract_address: String,
+    pub private_key: String,
+    pub rpc_sdk_url: Url,
+}
+
+pub fn load_enviroment_config() -> SdkConfig {
+    dotenv().ok();
+
+    let order_gateway_contract_address = env::var("ORDER_GATEWAY_CONTRACT_ADDRESS")
+        .expect("Order gateway contract address must be set as environment variable")
+        .to_lowercase();
+
+    let passiv_perp_contract_address = env::var("PASSIVE_PERP_GATEWAY_CONTRACT_ADDRESS")
+        .expect("Passive perp contract address must be set as environment variable")
+        .to_lowercase();
+
+    let private_key = env::var("PRIVATE_KEY")
+        .expect("Private key must be set as environment variable")
+        .to_lowercase();
+
+    let rpc_sdk_url = Url::parse(
+        env::var("RPC_SDK_URL")
+            .expect("Rpc Sdk Url must be set as environment variable")
+            .to_lowercase()
+            .as_str(),
+    );
+
+    let sdk_config = SdkConfig {
+        order_gateway_contract_address: order_gateway_contract_address,
+        passiv_perp_contract_address: passiv_perp_contract_address,
+        private_key: private_key,
+        rpc_sdk_url: rpc_sdk_url.unwrap(),
+    };
+    return sdk_config;
+}
 
 #[allow(dead_code)]
-pub static CORE_CONTRACT_ADDRESS: &str = "0xA763B6a5E09378434406C003daE6487FbbDc1a80";
+pub const REYA_EXCHANGE_ID: u128 = 1u128; //1=reya exchange
+pub const ETH_MARKET_ID: u32 = 1u32; //1=reya eth market
+pub const BTC_MARKET_ID: u32 = 2u32; //1=reya btc exchange
```

Comment:
2 =reya btc exchange? hmm, can you remind me why we need to hardcode these?

Reply:
We have one exchange, the reya exchange, and 2 markets, eth and btc

Reply:
yeah, just typo in comment 1=reya btc exchange should be 2=reya btc marjet